### PR TITLE
docs: fix data_dir config tip

### PR DESCRIPTION
**Pull request by zmstone**

### CODE REVIEW

File: en_US/configuration/configuration.md [hunk 1]
```diff
@@ -40,7 +40,12 @@ The configuration rewrite files are located in the `$data/configs/` directory, a
 | ---------------------------------- | -------------------- |
 | Installed with compression package | `./data`             |
 | Installed with binary package      | `/var/lib/emqx/data` |
```
Comment (id) [inline en_US/configuration/configuration.md]:
```suggestion
| Installed with binary package      | `/var/lib/emqx` |
```

Comment (id) [reply en_US/configuration/configuration.md]: while you are here

File: en_US/configuration/configuration.md [hunk 2]
```diff
@@ -6,15 +6,15 @@ Users can configure EMQX with configuration files or environment variables. This
...
-| Installed with binary package      | `/etc/emqx/emqx.conf`       |
-| Installed with compression package | `./etc/emqx.conf`         |
-| Installed with Docker              | `/opt/emqx/etc/emqx.conf` |
+| Installation                               | Path                      |
+| ------------------------------------------ | ------------------------- |
+| Installed with RPM or DEB package          | `/etc/emqx/emqx.conf`     |
+| Running in docker container                | `/opt/emqx/etc/emqx.conf` |
+| Extracted from portable compressed package | `./etc/emqx.conf`         |
 
 As the main configuration file, `emqx.conf` contains most of the commonly used configuration items. You can follow the example provided in the `emqx-example.conf` file (located within the same folder) to customize the settings. EMQX will use the default setting for other items to simplify your configuration work.
```
Comment (id) [inline en_US/configuration/configuration.md:22]: Down below
> EMQX also provides **two** configuration rewrite files implicitly nested under `emqx.conf`:

File: en_US/configuration/configuration.md [hunk 3]
```diff
@@ -6,58 +6,58 @@ Users can configure EMQX with configuration files or environment variables. This
...
 
-`emqx.conf` defines settings at a global level, for cases where you need to customize the settings for a cluster or a node, EMQX also provides two configuration rewrite files implicitly nested under `emqx.conf`:
+`emqx.conf` defines settings at a global level, for cases where you need to customize the settings for a cluster or a node,
+EMQX also provides a configuration rewrite file implicitly overlaying `emqx.conf`:
 
-- `cluster-override.conf`
+**`cluster-override.conf`**
 
-  Contains configuration items for the EQMX cluster. You can customize the settings with Dashboard,  REST API, and CLI. Note: Configurations in this file will overwrite configuration items with the same name in `emqx.conf`.
+Contains configuration items for entire cluster, configuration cahnges made from Dashboard, REST API, and CLI when be persisted to this file.
```
Comment (id) [inline en_US/configuration/configuration.md]:
```suggestion
Contains configuration items for the entire cluster, configuration changes made from Dashboard, REST API, and CLI will be persisted to this file.
```

**Review by id — APPROVED** [review]

**Review by LenaLenaPan — APPROVED** [review]

**Pull request merged**